sycl/CMakeLists.txt

```diff
@@ -435,6 +435,8 @@ set( SYCL_TOOLCHAIN_DEPLOY_COMPONENTS
      clang-offload-bundler
      clang-offload-deps
      clang-offload-extract
+     clang-offload-packager
```

Comment:
We need this to get the CI to package the files so they are usable in the tests here.